src/Middlewares/validateParams.ts

```diff
@@ -0,0 +1,17 @@
+import express, { Request, Response,NextFunction} from 'express'
+const middleware = (req: Request,res: Response, next:NextFunction) => {
+
+    const operation = req.query.op;
+    const a = req.query.n1;
+    const b = req.query.n2;
+    var nb1: number = +a;
+    var nb2: number = +b;
+
+  
+        if (operation === "mult" && a==0 ){
+            res.status(400).send("erreur")
```

Comment:
You should consider checking on the case if one of the variables `n1` or `n2` was undefined, in this case, your result can be NaN so please make sure always to handle all the cases here.